scripts/mean_guide

```diff
@@ -45,6 +46,11 @@ cp -r $TMP/meanFITS/"$1".fits $TMP/SDSS/G.in/
 /extra/wayne1/preserve/skylarjh/SpArcFiRe/scripts/SpArcFiRe -convert-FITS $TMP/SDSS/G.in $TMP/SDSS/G.tmp $TMP/SDSS/G.out -writeBulgeMask 1
```

Comment:
Oh wow. I hadn't seen this before... this *also* won't work anywhere except openlab... but that's been there for years. So maybe FOR NOW we can try this but it's a terrible hack that needs to be made more consistent to work anywhere, not just openlab.